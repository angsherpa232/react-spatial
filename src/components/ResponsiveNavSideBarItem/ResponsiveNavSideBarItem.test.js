// import React from 'react';
// import renderer from 'react-test-renderer';
// import { configure } from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16';
// import ResponsiveNavSideBarItem from './ResponsiveNavSideBarItem';

// configure({ adapter: new Adapter() });

// describe('ResponsiveNavSideBarItem', () => {
//   test('should match snapshot.', () => {
//     const component = renderer.create(<ResponsiveNavSideBarItem />);
//     const tree = component.toJSON();
//     expect(tree).toMatchSnapshot();
//   });
// });
