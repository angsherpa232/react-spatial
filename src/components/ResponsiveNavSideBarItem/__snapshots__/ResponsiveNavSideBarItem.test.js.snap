// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResponsiveNavSideBarItem should match snapshot. 1`] = `null`;